### PR TITLE
BDD scenarios for Sign in and restore password

### PR DESCRIPTION
**Pull request by anovojdion**

### CODE REVIEW

File: src/test/resources/feature/SignInScenarios.feature [hunk 1]
```diff
@@ -0,0 +1,32 @@
+@Anatolie
+Feature: Sign in and Forgot password
+
+  Scenario: Login validation with correct data
+    Given User is on "Authentication" page
```
Comment (VitalinaPetrenco) [inline src/test/resources/feature/SignInScenarios.feature:5]: If you have some initial steps that repeats all over the feature file (ex User is on "Authentication" page) you can move it to Background.

File: src/test/resources/feature/SignInScenarios.feature [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    When User introduce "test01@test.com" into "Email address" field
+    And User introduce "1@aBc" into "Password" field
+    And User clicks on "Sing in" button
+    Then User is redirected to "My account" page
+
+
+  Scenario Outline: Verify error handling (Negative scenario)
```
Comment (VitalinaPetrenco) [inline src/test/resources/feature/SignInScenarios.feature]: Could you please rename this scenario for more informative one (specify area where you are handling error)

File: src/test/resources/feature/SignInScenarios.feature [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+      | email         | password |
+      | test01@       |          |
+      |               |          |
+      | @             | 1        |
+      | test@test.com |          |
+
+
+  Scenario: Verify Retrieve Password button
+    Given User is on "Authentication" page
+    And User is redirected to "Forgot your password" page by clicking "forgot your password" link
```
Comment (VitalinaPetrenco) [inline src/test/resources/feature/SignInScenarios.feature]: Could you please separate this step for two in order to handle clicking and redirection functionality separately. For example:
And user clicks "forgot your password" link
And user is redirected to "Forgot your password" page

File: src/test/resources/feature/SignInScenarios.feature [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+    Examples:
+      | email         | password |
+      | test01@       |          |
+      |               |          |
+      | @             | 1        |
+      | test@test.com |          |
+
+
+  Scenario: Verify Retrieve Password button
```
Comment (VitalinaPetrenco) [inline src/test/resources/feature/SignInScenarios.feature]: If you are validating that correct confirmation message received so you can rename scenario to "Verify Retrieve Password confirmation message" to show which part functionality you are validating

File: src/test/resources/feature/SignInScenarios.feature [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+    And User introduce "1@aBc" into "Password" field
+    And User clicks on "Sing in" button
+    Then User is redirected to "My account" page
+
+
+  Scenario Outline: Verify error handling on Authentication page (Negative scenario)
+    When User introduce '<email>' into "Email address"
+    And User introduce '<password>' into "Password"
+    And User clicks on "Sing in" button
+    Then The error message is displayed
```
Comment (VitalinaPetrenco) [inline src/test/resources/feature/SignInScenarios.feature:18]: Could you please specify error message here.

File: src/test/resources/feature/SignInScenarios.feature (hunk 1 above)
Comment (anovojdion) [reply src/test/resources/feature/SignInScenarios.feature:5]: Done

**Pull request merged**